.gitignore

```diff
@@ -322,8 +322,8 @@ BenchmarkDotNet.Artifacts/
 tracer/deploy/linux/
 tracer/deploy/AzureAppServices/
 !tracer/build/artifacts
-tracer/src/Datadog.Monitoring.Distribution/home
-!tracer/src/Datadog.Monitoring.Distribution/home/**/*.txt
+tracer/src/Datadog.Trace.Bundle/home
+!tracer/src/Datadog.Trace.Bundle/home/**/*.txt
```

Comment:
You've deleted the readme.txt files so you might be able to delete this line

Reply:
Done in 05964fbfe0871542759b1d5d2cf1b742f6af58f6